NoComponentDrag.py

```diff
@@ -143,21 +135,21 @@ def is_parametric_mode():
         pass
     return False
 
+def clear_ui_item(item):
+    if item:
+        item.deleteMe()
+
 def run(context):
-    global app_
-    global ui_
-    global enable_cmd_def_
-    global select_panel_
-    global fusion_drag_controls_cmd_def_
+    #Expose global variables inside of function
+    global app_, ui_, enable_cmd_def_, select_panel_, fusion_drag_controls_cmd_def_
```

Comment:
Same concept with the global defs here: Simply exposing the already used variables to the function so seperate lines for each is pointless. If you were to create a global variable you could do so in another line seperate from the exposing line to improve readability.

Reply:
Comment is a bit redundant - after all, using "global" says exactly that code will be global.